internal/view/assets/js/page/setting.js

```diff
@@ -332,15 +318,12 @@ export default {
 					}
 
 					var request = {
-						username: account.username,
-						oldPassword: data.oldPassword,
-						newPassword: data.password,
-						owner: account.owner,
+						password: data.password,
 					};
 
 					this.dialog.loading = true;
-					fetch(new URL("api/accounts", document.baseURI), {
-						method: "put",
+					fetch(new URL(`api/v1/accounts/${account.id}`, document.baseURI), {
+						method: "patch",
```

Comment:
If you mean why using `api/v1/accounts/${account.id}` route instead of getting the ID from the Payload, that's how REST APIs work. You access to a specific resource by it's identifier, and execute actions (verbs: GET, POST, PATCH, DELETE) over it. It's common practice. This way just by knowing the URL to a resource you only need to switch the verb to perform actions over it.